README.md

```diff
@@ -2,4 +2,24 @@
 
 In a series of articles we document how we build a distributed hospitality exchange over [Solid](https://solidproject.org).
 
-[Start here](intro.md)
+[comment]: <> (Who are we? Would be nice to write that it was you together with OHN project/team or you as part of it)
+
+[Start here](intro.md)...
+
+---
+[comment]: <> (I enjoyed the mystery of unknowing "what happened next" but I believe many people may prefer to know what to expect before starting to read, so you may consider to add an overview here)
+... or jump over directly to one of the chapters:
+
+1. [Developing a distributed app with Solid](intro.md) \
```

Comment:
I love this table of contents. Definitely should be included 🎉 
And a perfect content for the intro site.

Point 8 is optional reading, so i'll change the numbering to 7.1 (Optional)

Reply:
I left it as 8, because it's hard to [make 7.1 in markdown](https://meta.stackexchange.com/questions/85474/how-to-write-nested-numbered-lists)

Reply:
published in a0e413f19c3